client/state/action-types.js

```diff
@@ -429,6 +429,7 @@ export const INLINE_HELP_SEARCH_RESET = 'INLINE_HELP_SEARCH_RESET';
 export const INLINE_HELP_SEARCH_REQUEST_FAILURE = 'INLINE_HELP_SEARCH_REQUEST_FAILURE';
 export const INLINE_HELP_SEARCH_REQUEST_SUCCESS = 'INLINE_HELP_SEARCH_REQUEST_SUCCESS';
 export const INLINE_HELP_SEARCH_REQUEST_API_RESULTS = 'INLINE_HELP_SEARCH_REQUEST_API_RESULTS';
+export const INLINE_HELP_SET_QUERY_SEARCH = 'INLINE_HELP_SET_QUERY_SEARCH';
```

Comment:
```suggestion
export const INLINE_HELP_SET_SEARCH_QUERY = 'INLINE_HELP_SET_SEARCH_QUERY';
```

The thing being "set" is the "search query" not the "query search". I'm suggesting this as I think it will read better in English. Feel free to tell me if I've misunderstood.

Reply:
Note that the property of the state is also `searchQuery` and not `querySearch` so it makes sense to align with this for consistency.

Reply:
THis action is still named incorrectly. It should be `INLINE_HELP_SET_SEARCH_QUERY`. This will bring it into line with the other changes you've made to correct this.